concepts/extensions/elasticsearch-concept.md

```diff
@@ -0,0 +1,52 @@
+# Elasticsearch
+
+Elasticsearch is a NoSQL Database focused on search capabilities to act as a search engine.
+The Shopware implementation of Elasticsearch provides an integrated way to improve the performance of product and category searches.
+To use Elasticsearch for your shop take a look at our [elasticsearch guide](guides/hosting/elasticsearch)
+
+## Concept
```

Comment:
I mention adding also the addState (https://github.com/shopware/platform/blob/trunk/src/Elasticsearch/Framework/ElasticsearchHelper.php#L117). Otherwise the search requests will go still to SQL.